### DISCUSSION TITLE
Can't get past Setting up workspace

### DISCUSSION THREAD

**Discussion by LeoMaguire** [Budibase/budibase#12061] [Help] [answered]
Either gives a Error: User xxx@xxxxxx already exists.  (xxx are really my email)

and no matter what I try the login screen always redirects to the getting started wizard that returns the above error.

tried 2 browsers, private mode on 1 browser, multiple different email addresses, multiple different company names, tried using microsoft account for the login.

**Comment by LeoMaguire** [comment]
well - I just now got a confirmation email for 1 of the emails I attempted to use - but there still seem to be some issues.

Overall - this getting signed up process seems to be a mess - is this normal?  Why can't I just log in to your site with my MS creds and not get "this account already exists" errors without any way to try to log in to this account?

**Reply by ConorWebb96** [reply, marked as answer]
Hey @LeoMaguire,

We are aware that there is an [issue](https://github.com/Budibase/budibase/issues/12052) with Microsoft SSO. For now can you email into `support@budibase.com` with the emails used to create the account and then we will delete them for you. An engineer has been assigned to the issue and is currently working on a potential fix.

Once this has been done please use the email option to create your account. After the fix has been implemented you can simply sign in with SSO and the account type will change from password to SSO. 

Apologies for the inconvenience and I hope this helps.

**Comment by LeoMaguire** [comment]
You guys appear to have larger problems with your authorization systems – now I can’t even change my password used with an email address.

Using reset password – link arrives – no matter what I do, when I enter a new password I get a generic “Unable to reset password” message

From: Conor Webb ***@***.***>
Date: Monday, October 16, 2023 at 3:53 AM
To: Budibase/budibase ***@***.***>
Cc: Leo Maguire ***@***.***>, Mention ***@***.***>
Subject: Re: [Budibase/budibase] Can't get past Setting up workspace (Discussion #12061)

Hey @LeoMaguire<https://github.com/LeoMaguire>,

We are aware that there is an issue<https://github.com/Budibase/budibase/issues/12052> with Microsoft SSO. For now can you email into ***@***.*** with the emails used to create the account and then we will delete them for you. An engineer has been assigned to the issue and is currently working on a potential fix.

Once this has been done please use the email option to create your account. After the fix has been implemented you can simply sign in with SSO and the account type will change from password to SSO.

Apologies for the inconvenience and I hope this helps.

**Reply by ConorWebb96** [reply]
Hey @LeoMaguire,

I was able to reset a forgotten password last night and again this morning. Where you perhaps trying to reset a password of an account you created with SSO? (Basing this purely off the thread above.)

Feel free to send an email to support@budibase.com providing the email address that you were trying to change the password of and I'll have a look at it.

Thanks in advance.

**Comment by LeoMaguire** [comment]
No – SSO using MS doesn’t work so I used a personal email address.

Just tried resetting it again – user account is ***@***.******@***.***> – and here is an example of one of the passwords I’m trying to use.    juct0och!nad1OO

Still same issue – can’t log in, click reset, I get the email, click the link, choose a password, get the error message below with no other indication of what the issue may be.

[A screenshot of a computer screen  Description automatically generated]

From: Conor Webb ***@***.***>
Date: Thursday, October 19, 2023 at 3:07 AM
To: Budibase/budibase ***@***.***>
Cc: Leo Maguire ***@***.***>, Mention ***@***.***>
Subject: Re: [Budibase/budibase] Can't get past Setting up workspace (Discussion #12061)

Hey @LeoMaguire<https://github.com/LeoMaguire>,

I was able to reset a forgotten password last night and again this morning. Where you perhaps trying to reset a password of an account you created with SSO? (Basing this purely off the thread above.)

Feel free to send an email to ***@***.******@***.***> providing the email address that you were trying to change the password of and I'll have a look at it.

Thanks in advance.

**Comment by LeoMaguire** [comment]
Also – my wife still has her access – she tried to send an invitation to a never used email address and when trying to accept the invitation I get this.


{"message":"Name: ***@***.***_global-db'. Only lowercase characters (a-z), digits (0-9), and any of the characters _, $, (, ), +, -, and / are allowed. Must begin with a letter.","status":400}


From: Leo Maguire ***@***.***>
Date: Thursday, October 19, 2023 at 7:50 AM
To: Budibase/budibase ***@***.***>, Budibase/budibase ***@***.***>
Cc: Leo Maguire ***@***.***>, Mention ***@***.***>
Subject: Re: [Budibase/budibase] Can't get past Setting up workspace (Discussion #12061)
No – SSO using MS doesn’t work so I used a personal email address.

Just tried resetting it again – user account is ***@***.******@***.***> – and here is an example of one of the passwords I’m trying to use.    juct0och!nad1OO

Still same issue – can’t log in, click reset, I get the email, click the link, choose a password, get the error message below with no other indication of what the issue may be.

[A screenshot of a computer screen  Description automatically generated]

From: Conor Webb ***@***.***>
Date: Thursday, October 19, 2023 at 3:07 AM
To: Budibase/budibase ***@***.***>
Cc: Leo Maguire ***@***.***>, Mention ***@***.***>
Subject: Re: [Budibase/budibase] Can't get past Setting up workspace (Discussion #12061)

Hey @LeoMaguire<https://github.com/LeoMaguire>,

I was able to reset a forgotten password last night and again this morning. Where you perhaps trying to reset a password of an account you created with SSO? (Basing this purely off the thread above.)

Feel free to send an email to ***@***.******@***.***> providing the email address that you were trying to change the password of and I'll have a look at it.

Thanks in advance.
